idc_index/index.py

```diff
@@ -238,10 +259,19 @@ def fetch_index(self, index) -> None:
                     idc_index_data.IDC_INDEX_PARQUET_FILEPATH.parents[0],
                     f"{index}.parquet",
                 )
+
                 with open(filepath, mode="wb") as file:
                     file.write(response.content)
-                setattr(self.__class__, index, pd.read_parquet(filepath))
+
+                # Join new index with main index
```

Comment:
I would think that here we just need the `series_aws_url` column from `index`, no?

Reply:
We would also need all the attributes necessary for building the hierarchy, but I fixed this in the respective SQL query and do not merge anything when adding a new index. 
I am wondering why checks passed for commit [bbef810](https://github.com/ImagingDataCommons/idc-index/pull/113/commits/bbef810a398a6cb9923997daca37ffb318d82c07), where they shouldn't in my opinion and also did not when I executed them manually.